packages/core/test/tabs/tabsTests.tsx

```diff
@@ -113,7 +115,10 @@ describe("<Tabs>", () => {
                 <Tab id="third" title="Third" className={tabClassName} panel={<Panel title="third" />} />,
             </Tabs>,
         );
-        assert.lengthOf(wrapper.find(`.${tabClassName}`), 9);
+        const NUM_TABS = 3;
+        assert.lengthOf(wrapper.find(TAB_SELECTOR), NUM_TABS);
+        assert.lengthOf(wrapper.find(TAB_PANEL_SELECTOR), NUM_TABS);
+        assert.lengthOf(wrapper.find(`.${tabClassName}`).hostNodes(), NUM_TABS * 2);
```

Comment:
Should not have length 9, should have length 6. Need to use `hostNodes` to remove the React nodes.